draft-ietf-dmarc-aggregate-reporting.md

```diff
@@ -276,6 +272,8 @@ following:
    single "policy_published" element;
 *  generate multiple reports for the same period, one for each
    published policy occurring during the reporting period;
+*  generate a report whose end time occurs when the updated policy was 
+   detected, regardless of reporting intervals.
```

Comment:
Can this wording be misinterpreted to mean that <begin> and <end> don't have to be exact?  Perhaps it is clearer to say something like *regardless of the previously planned interval* or something like that.